internal/cli/serve.go

```diff
@@ -24,23 +27,69 @@ func runServe(cmd *cobra.Command, cmdArgs []string) error {
 		"groups": runSettings.LockGroups,
 	}).Debug("lock groups")
 
-	logrus.WithFields(logrus.Fields{
-		"address": runSettings.ServiceAddress,
-		"port":    runSettings.ServicePort,
-	}).Info("starting service")
-
 	if runSettings == nil {
 		return errors.New("nil runSettings")
 	}
 	airlock := server.Airlock{*runSettings}
 
-	http.Handle(server.PreRebootEndpoint, airlock.PreReboot())
-	http.Handle(server.SteadyStateEndpoint, airlock.SteadyState())
+	stopCh := make(chan os.Signal)
+	signal.Notify(stopCh, os.Interrupt)
```

Comment:
`signal.Notify` takes a channel and `... os.Signal` so we should be able to combine these lines, no?